test/slots/Slot.test.js

```diff
@@ -28,33 +34,28 @@ describe('the Slot class', () => {
   })
 
   describe('.constructor', () => {
-    describe('on InSlots', () => {
+    describe('on FloatInSlots', () => {
       it('sets tareable to false by default', () => {
-        const slotData = InSlotSeeder.generate()
+        const slotData = FloatInSlotSeeder.generate()
         delete slotData.tareable
 
-        const slot = new InSlot(slotData)
+        const slot = new FloatInSlot(slotData)
 
         expect(slot.tareable).toBe(false)
       })
+    })
 
-      it('sets defaultValue to null by default', () => {
-        const slotData = InSlotSeeder.generate()
-        delete slotData.defaultValue
+    describe('on IntegerInSlots', () => {
+      it('sets tareable to false by default', () => {
+        const slotData = IntegerInSlotSeeder.generate()
+        delete slotData.tareable
 
-        const slot = new InSlot(slotData)
+        const slot = new IntegerInSlot(slotData)
 
-        expect(slot.defaultValue).toBeNull()
+        expect(slot.tareable).toBe(false)
       })
     })
```

Comment:
im not 100% sure about the following comment:

I believe `InSlot` no longer has anything to do with `tareable`
Both of these seem to actually be testing behavior the `Float/IntegerInSlot.constructor` are doing, and not the `Slot.constructor`.

Reply:
this is correct, I forgot to move those to the particular single slot test file

Reply:
fixed